src/utilities/get-issue-numbers.ts

```diff
@@ -0,0 +1,23 @@
+export function getIssueNumbers(text: string): number[] {
+  let processedString = text;
+  if (processedString.includes('>')) {
+    processedString = processedString.replace(/>(?<=>)(.*)(?=\n)/g, '');
+  }
+  const issueStrings = processedString.match(/#[0-9]+/g);
+
+  const issueNumbers: number[] = [];
+
+  if (issueStrings) {
+    issueStrings.forEach((issue) => {
+      const issueNumberOrNaN = Number(issue.substr(1));
+
+      if (!isNaN(issueNumberOrNaN)) {
+        if (issueNumberOrNaN > 50) {
+          issueNumbers.push(issueNumberOrNaN);
+        }
+      }
+    });
+  }
```

Comment:
So I peeked at the tests you were running - this code block means if I do `#100 #101 #102` the bot will send three links?

Reply:
Yes. But I don't think anybody will do that tbh.